libcnb-test/tests/integration_test.rs

```diff
@@ -522,3 +541,42 @@ fn address_for_port_when_container_crashed() {
         },
     );
 }
+
+#[test]
+#[ignore = "integration test"]
+fn basic_build_with_libcnb_reference_to_single_buildpack() {
+    TestRunner::default().build(
+        BuildConfig::new("heroku/builder:22", "test-fixtures/procfile").buildpacks(vec![
```

Comment:
Since the test buildpacks always pass detection and don't use the procfile, they can use the empty fixture (using the procfile fixture but not the procfile buildpack made me wonder if there was something special about the procfile that meant it was still needed).

```suggestion
        BuildConfig::new("heroku/builder:22", "test-fixtures/empty").buildpacks(vec![
```